com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Deferred/RaytracingGBuffer.raytrace

```diff
@@ -116,6 +116,9 @@ void TraceGBuffer(PixelCoordinates coords)
     rayDescriptor.Direction = direction.xyz;
     rayDescriptor.TMin = 0.0;
     rayDescriptor.TMax = _RaytracingRayMaxLength;
+    
+    // Adjust world-space position to match the RAS setup with XR single-pass and camera relative
+    ApplyCameraRelativeXR(rayDescriptor.Origin);
```

Comment:
@anisunity / @fabien-unity  by doing call like this we will always make mistake with copy/paste 

As far as I can see the code is always the same. We should do a function and call it.

RayDesc rayDescriptor;
InitRayDesc(rayDescriptor, posInput.positionWS, normalData.normalWS, direction);

with 

InitRayDesc(RayDesc  rayDescriptor, ...)
{
    rayDescriptor.Origin = ApplyCameraRelativeXR(posInput.positionWS + normalData.normalWS * _RaytracingRayBias);
    rayDescriptor.Direction = direction.xyz;
    rayDescriptor.TMin = 0.0;
    rayDescriptor.TMax = _RaytracingRayMaxLength;
}

Thought?

Reply:
Yes that'll be ideal. I can refactor the code, unless @anisunity wants to do it :)

Reply:
i'll Do it. Does it have to be in this PR?

Reply:
@fabien-unity

Reply:
I'm ok with the refactor being done after this PR lands.

Reply:
alright, landing this PR then :)